src/ExpressionEvaluator/Core/Source/ResultProvider/ResultProviderPhone.csproj

```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <OutDirSuffix>Phone\</OutDirSuffix>
+    <ProjectGuid>{6A4C8A1D-4C47-4191-9681-136FC3B7209C}</ProjectGuid>
```

Comment:
Since you're including the project after the GUID is set, the other project will overwrite this.